racket-poppler/render-tex.rkt

```diff
@@ -44,15 +44,15 @@
   (set! defaultdir
         (or defaultdir
             (case (system-type)
-              [(macosx) "/Library/TeX/texbin"]
+              [(macosx) "/usr/texbin"]
```

Comment:
This is tricky: From "UpdatingForElCapitan.pdf" 

>  When you install MacTEX-2015 it not only creates /usr/texbin, if allowed for the present OS
> version, but also creates a second symbolic link, /Library/TeX/texbin, in /Library/TeX which
> is “owned” by MacTEX so is allowed by El Capitan.

I read this as "/usr/texbin/" is only available on some systems - whereas /Library/Tex/texbin is available only on some systems. I (very) recently installed TexLive 2015 and my /usr looks likes this:

> ls /usr
> X11 X11R6 adic bin include		
> lib libexec local sbin share standalone

Do you have both?